.travis.yml

```diff
@@ -138,6 +138,9 @@ jobs:
     - stage: Py38
       name: "Python 3.8"
       python: 3.8
+    - stage: Py310
+      name: "Python 3.10"
+      python: 3.10
```

Comment:
is this temporary or final?

Reply:
If we run the tests sequentially: yes this will be final

Reply:
Check it out https://app.travis-ci.com/github/radis/radis/builds/274479051

Reply:
What triggered my comment is I was seing a 3rd Python version added. Actually, the issue is there is this Python 3.8 still there. If 3.10 is already Vaex-compatible; we could remove 3.8 ?

Reply:
As long as it passes on the master for a new version, I'd like to keep it. If one day it raises problems, let's remove it. I could add a comment next to it

Reply:
ok